doc/nrf/known_issues.rst

```diff
@@ -1849,6 +1924,35 @@ tx_buffer_length set incorrectly
 Trusted Firmware-M (TF-M)
 *************************
 
+.. rst-class:: v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-11195: Build errors when enabling CONFIG_BUILD_WITH_TFM option
+  Configuring the BUILD_WITH_TFM configuration in SES project configuration or using ``west -t menuconfig`` results in build errors.
+
+  **Workaround:** Set ``CONFIG_BUILD_WITH_TFM=y`` in project configuration file (:file:`prj.conf`) or through west command line (``west build – -DCONFIG_BUILD_WITH_TFM=y``).
+
+.. rst-class:: v1-9-0
+
+NCSDK-13530: TF-M minimal build has increased in size
+  TF-M minimal build exceeds 32 kB due to increased dependencies in the TF-M crypto partition.
+
+.. rst-class:: v1-9-0
+
+KRKNWK-12777: FLASHACCER event triggered after soft reset
+  After soft reset, TF-M sometimes triggers a FLASHACCERR event and execution halts.
+
+.. rst-class:: v1-9-0 v1-8-0
+
+NCSDK-12342: SecureFault exception while accessing protected storage
+  When accessing protected storage, a SecureFault exception is sometimes triggered and execution halts.
+
+.. rst-class:: v1-9-0 v1-8-0
+
+NCSDK-12483: Missing debug symbols
+  Some debug symbols are missing sometimes.
+
+  **Workaround:** Add the following text in the application :file:`CMakeLists.txt` file: ``zephyr_link_libraries(-Wl,--undefined=my_missing_debug_symbol)``.
+
 .. rst-class:: v1-8-0
 
 NCSDK-10022: Missing AES-CFB support in Nordic Security Module
```

Comment:
We would like to update this known issue, which was not concise enough.

```suggestion
.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0

NCSDK-10022: TF-M Regression tests fails on AES-CFB tests.
TF-M Regression tests fails on AES-CFB tests because AES-CFB is not supported in :ref:`nrfxlib:nrf_security` (nrf_security).
```

Remove the rest of the description.

Reply:
Updated

Reply:
Sorry for the mess, there is a bookkeeping error.
This does not affect any releases, and therefore should not be in the known-issues.
Please remove this entry.

Reply:
No problem! Removed the known issue.